dask/array/core.py

```diff
@@ -2660,6 +2660,10 @@ def bincount(x, weights=None, minlength=None):
 
     return Array(dsk, name, chunks, dtype)
 
+@wraps(np.digitize)
+def digitize(a, bins, right=False):
+    func = partial(np.digitize, bins=bins, right=right)
+    return elemwise(func, a)
```

Comment:
In this case I recommend `a.map_partitions(np.digitize, bins=bins, right=right)`

This avoids the partial (which can be slightly more costly to serialize in a distributed setting) and should work just as well.